utilities/cluster_config_utils.py

```diff
@@ -87,3 +87,21 @@ def is_xattrs_enabled(cluster_config):
 
     cluster = load_cluster_config_json(cluster_config)
     return cluster["environment"]["xattrs_enabled"]
+
+
+def is_load_balancer_enabled(cluster_config):
+    """ Loads cluster config to see if load balancer is enabled """
+    cluster = load_cluster_config_json(cluster_config)
+    return cluster["environment"]["sg_lb_enabled"]
+
+
+def get_load_balancer_ip(cluster_config):
+    """ Loads cluster config to fetch load balancer ip """
+    cluster = load_cluster_config_json(cluster_config)
+
+    num_lbs = len(cluster["load_balancers"])
+    if num_lbs < 1:
```

Comment:
This should be strict == 1, since you are referencing that specific index. It could be confusing with multiple lbs